src/plugins/bubble/_base.scss

```diff
@@ -0,0 +1,432 @@
+		-webkit-animation-timing-function: ease-out;
+			animation-timing-function: ease-out;
+		-webkit-transform-origin: 100% 50%;
+			transform-origin: 100% 50%;
+		will-change: scroll-position; }
+
+.trans-pulse {
+		-webkit-animation: .5s linear 3.5s 1 pulseIn, .5s linear 15s 1 pulseIn, .5s linear 30s 1 pulseIn;
+			animation: .5s linear 3.5s 1 pulseIn, .5s linear 15s 1 pulseIn, .5s linear 30s 1 pulseIn;
+		will-change: transform; }
+
+.loader-typing {
+	bottom: 30%;
+	height: 6px;
+	left: 30px;
+	position: absolute;
+	-webkit-transform: translateX(-50%) translateY(-50%);
+			transform: translateX(-50%) translateY(-50%);
+	width: 26px; }
+
+.loader-dot {
+	-webkit-animation: grow .5s ease-in-out infinite alternate;
+			animation: grow .5s ease-in-out infinite alternate;
+	background-color: #444;
+	border-radius: 50%;
+	height: 6px;
+	position: absolute;
+	width: 6px;
+	will-change: transform; }
+
+.loader-dot.dot1 {
+	left: 0;
+	-webkit-transform-origin: 100% 50%;
+			transform-origin: 100% 50%; }
+
+.loader-dot.dot2 {
+	-webkit-animation-delay: .1s;
+			animation-delay: .1s;
+	left: 50%;
+	margin-left: -3px;
+	-webkit-transform: scale(0.99);
+			transform: scale(0.99); }
+
+.loader-dot.dot3 {
+	-webkit-animation-delay: .2s;
+			animation-delay: .2s;
+	right: 0; }
+
+button.wb-sl {
+	padding: 5px;
+	z-index: 501;
+	width: auto !important;
+	background: #26374a;
+	font-weight: 700;
+	border: none;
+	cursor: pointer;
+}
+/* Chat wizard Bubble */
+.wb-bubble-bubble-wrap {
+	bottom: 30px;
+	height: 60px;
+	position: fixed;
+	right: 30px;
+	width: 60px;
+	z-index: 1049; }
+
+.wb-bubble-bubble-wrap p {
+	background: #335075;
+	border-bottom-left-radius: 25px;
+	border-top-left-radius: 25px;
+	-webkit-box-shadow: 0 1px 3px rgba(0, 0, 0, 0.45);
+			box-shadow: 0 1px 3px rgba(0, 0, 0, 0.45);
+	color: #fff;
+	font-size: 14px;
+	line-height: 20px;
+	min-height: 50px;
+	padding: 5px 37.5px 5px 27.5px;
+	position: relative;
+	right: 195px;
+	top: 5px;
+	width: 225px; }
+
+.wb-bubble-bubble-wrap p .notif-close {
+	background: #333;
+	border-radius: 50%;
+	color: #fff;
+	font-size: 19px;
+	height: 1.25em;
+	line-height: 21px;
+	position: absolute;
+	right: 92.5%;
+	text-align: center;
+	text-decoration: none;
+	top: 0;
+	width: 1.25em; }
+
+.wb-bubble-bubble-wrap .notif {
+	cursor: pointer; }
+
+.wb-bubble-bubble-wrap .bubble {
+	background: #fff url("../assets/default-avatar.png") center no-repeat;
+	border-radius: 50%;
+	bottom: 0;
+	-webkit-box-shadow: 0 2px 4px rgba(0, 0, 0, 0.45);
+			box-shadow: 0 2px 4px rgba(0, 0, 0, 0.45);
+	height: 100%;
+	overflow: hidden;
+	position: absolute;
+	right: 0;
+	text-indent: -9999px;
+	white-space: nowrap;
+	width: 100%;
+	z-index: 1048; }
+
+.wb-bubble-bubble-wrap .bubble:focus {
+	border: 1px solid rgba(0, 0, 0, 0.5);
+	-webkit-box-shadow: 0 2px 3px rgba(0, 0, 0, 0.7);
+			box-shadow: 0 2px 3px rgba(0, 0, 0, 0.7); }
+
+.wb-bubble-btn-extrnl + .wb-bubble-bubble-wrap {
+	display: none !important; }
+
+/* Chat Wizard */
```

Comment:
Remove or replace this comment

Is the CSS bellow are only applicable to the bubble?